content/help/system-requirements.adoc

```diff
@@ -82,25 +82,61 @@ The software prerequisites for installing on a macOS system are as follows:
 [%hardbreaks]
 === GNU/Linux
 
-The following Linux distribution versions have been tested and are known to be working.
+The following Linux distributions have been tested and are known to be working with
+KiCad version 5.1.
 Issues or bugs that occur on an unsupported platform must be reproduced on an officially
 supported distribution and window manager before they will be addressed by KiCad.
 
-==== Tested Distributions
+*Note:* Development versions of the next KiCad release (version 6) may not function
```

Comment:
De we need to be so specific as to mention v6?

Reply:
Same rationale as why I said 5.1 instead of stable. I was afraid that saying just "development" might lead people to think the 5.1 nightly testing builds may not function (and if those don't work, we really do want to know because they should and they need to be fixed).